cves/CVE-2013-0843.yml

```
@@ -4,21 +4,21 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE: 
+CWE: CWE-787
```

The CWE value should just be the CWE ID number.

```suggestion
CWE: 787
```